cmd/vclusterctl/cmd/platform/backup.go

```diff
@@ -0,0 +1,135 @@
+package platform
+
+import (
+	"fmt"
+	"os"
+
+	storagev1 "github.com/loft-sh/api/v4/pkg/apis/storage/v1"
+	"github.com/loft-sh/api/v4/pkg/product"
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/log/survey"
+	"github.com/loft-sh/vcluster/pkg/cli/config"
+	"github.com/loft-sh/vcluster/pkg/cli/flags"
+	"github.com/loft-sh/vcluster/pkg/platform"
+	"github.com/loft-sh/vcluster/pkg/platform/backup"
+	"github.com/loft-sh/vcluster/pkg/platform/clihelper"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/clientcmd"
+	clientpkg "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	scheme = runtime.NewScheme()
+
+	_ = clientgoscheme.AddToScheme(scheme)
+	_ = storagev1.AddToScheme(scheme)
+)
+
+// BackupCmd holds the cmd flags
+type BackupCmd struct {
+	*flags.GlobalFlags
+	Log       log.Logger
+	Namespace string
+	Filename  string
+	Skip      []string
+	cfg       *config.CLI
+}
+
+// newBackupCmd creates a new command
+func newBackupCmd(globalFlags *flags.GlobalFlags, cfg *config.CLI) *cobra.Command {
+	cmd := &BackupCmd{
+		GlobalFlags: globalFlags,
+		Log:         log.GetInstance(),
+		cfg:         cfg,
+	}
+
+	description := product.ReplaceWithHeader("backup", `
```

Comment:
In the issue, the command looks to be `vcluster platform backup management`, however this would implement `vcluster platform backup`. Is that an intentional change?

Reply:
no, I updated it to to it like it is described in the ticke